x-pack/plugins/security_solution/server/assistant/tools/defend_insights/workflow_insights_builders/incompatible_antivirus.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+
+import { ENDPOINT_ARTIFACT_LISTS } from '@kbn/securitysolution-list-constants';
+
+import type { DefendInsight } from '@kbn/elastic-assistant-common';
+import type { BuildWorkflowInsightParams } from '.';
+import type { SecurityWorkflowInsight } from '../../../../../common/endpoint/types/workflow_insights';
+
+import {
+  ActionType,
+  Category,
+  SourceType,
+  TargetType,
+} from '../../../../../common/endpoint/types/workflow_insights';
+import { SUPPORTED_HOST_OS_TYPE } from '../../../../../common/endpoint/constants';
+
+export function buildIncompatibleAntivirusWorkflowInsight(
```

Comment:
nit: name should rather be `buildIncompatibleAntivirusWorkflowInsights` with plural at the end